tools/spec-gen-sdk/src/automation/reportStatus.ts

```diff
@@ -146,6 +147,133 @@ export const saveFilteredLog = async (context: WorkflowContext) => {
   context.logger.log('endsection', 'Save filtered log status');
 };
 
+export const renderFilterLog2HTML = async (context: WorkflowContext) => {
+    context.logger.log('section', 'Save filtered log HTML');
+    const RegexMarkdownSplit = /^(.*?)(<ul>.*)$/s;
+    const RegexNoteBlock = /> \[!NOTE\]\s*>\s*(.*)/;
+    const messageRecord = fs.readFileSync(context.filteredLogFileName).toString();
+    const parseMssageRecord = JSON.parse(messageRecord) as MessageRecord[];
+
+    let totalBody = '';
+    for(let commentBody of parseMssageRecord) {
+        const markdown = commentBody.message || '';
+        let blockString = '';
+        let bodyString = '';
+        
+        const match = markdown.match(RegexMarkdownSplit);
+        if (match !== null) { 
+            bodyString = match[2].trim(); 
+            const noteBlock = match[1].trim();
+            const noteMatch = noteBlock.match(RegexNoteBlock);
+            if (noteMatch !== null) {
+                blockString = noteMatch[1].trim();
+            }
+        } else {
+            bodyString = marked(markdown) as string;
+        }
+        const renderNoteBlock = blockString && htmlBlockTemp(blockString);
+        totalBody += (renderNoteBlock + bodyString);
+    }
+    
+
+    const htmlString: string = htmlTemp(totalBody);
+    
+    context.logger.info(`Writing html to ${context.logHtml}`);
+    fs.writeFileSync(context.logHtml, htmlString, "utf-8");
+    context.logger.log('endsection', 'Save filtered log HTML');
+}
+
+function htmlTemp(bodyString:string):string {
+    const githubStylesheet = "https://cdnjs.cloudflare.com/ajax/libs/github-markdown-css/5.2.0/github-markdown.min.css";
+    return `
+<!DOCTYPE html>
+<html lang="zh-CN">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Script Log Html</title>
+    <link rel="stylesheet" href="${githubStylesheet}">
+    <style>
```

Comment:
Can this inline style be loaded from an external file?

Reply:
as this html will publish as a artifact, it will render by a signle file! Uncessary for a external css file!